enterprise/models.py

```diff
@@ -107,3 +110,56 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+def logo_path(instance, filename):
+    """
+    Delete the file if it already exist and returns the enterprise customer logo image path.
+
+    :param instance: EnterpriseCustomerBrandingConfiguration object
+    :param filename: file to upload
+    :return path: path of image file e.g. enterprise_logo/1/filename
+    """
+    name = os.path.join('enterprise_logo', str(instance.id), filename)
+    fullname = os.path.join(settings.MEDIA_ROOT, name)
+    if os.path.exists(fullname):
+        os.remove(fullname)
+    return name
+
+
+class EnterpriseCustomerBrandingConfiguration(TimeStampedModel):
+    """
+    Model that keeps track of enterprise branding configurations e.g. enterprise customer logo.
+
+    Fields:
+        enterprise_customer (ForeignKey[EnterpriseCustomer]): enterprise customer
+        logo (ImageField): enterprise customer image
+    """
+
+    enterprise_customer = models.OneToOneField(
+        EnterpriseCustomer,
+        blank=False,
+        null=False,
+        related_name='branding_configuration'
+    )
+    logo = models.ImageField(
+        upload_to=logo_path,
```

Comment:
Aren't we planning to sync these `EnterpriseCustomer*` models in the `enterprise` app among different IDAs, such as Otto and the LMS? If so, syncing image files may be a pain. It may make more sense to upload the logos to something like S3, then just store a URL on the model. Or, if only the "source of truth" holds the EnterpriseCustomerBrandingConfiguration data, and others just access the logo via a URL to the source of truth app's image URL, that's fine, but should be explicitly modelled/explained in the code.

Also genuine, possibly related question: What's the advantage of making this a one-to-one model rather than a field on `EnterpriseCustomer`?